src/lib/.eslintrc.js

```diff
@@ -1,9 +1,11 @@
 module.exports = {
   rules: {
     "no-var": "off",
-    "func-names": "off"
+    "func-names": "off",
+    "import/no-default-export": 2,
+    "import/no-named-export": 2
   },
   globals: {
     turbine: "readonly"
   }
-};
+};
```

Comment:
You might want to configure your IDE/editor to insert a line break at the end of files or we might end up with unnecessary git churn.

Reply:
Done